apps/web-giddh/src/app/financial-reports/components/profit-loss/components/profit-loss-grid/components/profit-loss-grid-row/profit-loss-grid-row.component.html

```
@@ -78,8 +88,20 @@
+            <div
+                class="col-4 pl-sticky-group d-flex align-items-center row-height"
+                [style.padding-left]="padding + 20 + 'px !important'"
+            >
+                <mat-checkbox
+                    class="mr-l05"
+                    color="primary"
+                    (change)="onItemChecked($event, account?.uniqueName, 'account')"
+                    [(ngModel)]="account.checked"
+                    [checked]="account?.checked"
+                    (click)="$event.stopPropagation()"
+                    class="tailed-checkbox"
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ class ] was found.

(attr-no-duplication)